ui/src/app/editor/dialog/help/help.component.html

```diff
@@ -16,6 +16,29 @@
   ~
   -->
 
+<style>
+    /* .table {
+        margin: 20px 0;
+        font-size: 0.9em;
+        font-family: sans-serif;
+        min-width: 400px;
+        box-shadow: 0 0 20px rgba(0, 0, 0, 0.15);
+    } */
+
+    th, td {
+        padding: 15px;
+        text-align: left;
+        border-bottom: 1px solid #ddd;
+    }
+
+    .table tbody tr:hover {
+        font-weight: bold;
+        background-color: #f1ecec;
+    }
+
+    
+</style>
+
```

Comment:
Styles should be added to the component's scss file (see help.component.scss) and not directly in the HTML. Style tags are typically used in the HTML header section only.

See https://angular.io/guide/component-styles#style-files-in-component-metadata for some info on how styles can be included in Angular components.